packages/carbon-component-mapper/src/tests/text-field.test.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+
+import { FormRenderer, componentTypes } from '@data-driven-forms/react-form-renderer';
+
+import FormTemplate from '../form-template';
+import componentMapper from '../component-mapper';
+import { TextInput, NumberInput } from 'carbon-components-react';
+
+describe('<TextInput and NumberInput />', () => {
+  let onChange;
+  let wrapper;
+  beforeEach(() => {
+    onChange = jest.fn();
```

Comment:
`onChange` is not being used - and you should not override onChange in the schema if you have not a really special case.